tests/framework/installer/ceph_installer.go

```diff
@@ -484,6 +504,10 @@ func (h *CephInstaller) checkCephHealthStatus(namespace string) {
 
 	// Depending on the tests, the health may be fluctuating with different components being started or stopped.
 	// If needed, give it a few seconds to settle and check the status again.
+
+	if clusterResource.Status.CephStatus == nil {
```

Comment:
Do you have a log from a test that failed on this? I wouldn't expect the status to be nil at the end of a test. How about a `require.NotNil(h.T(), clusterResource.Status.CephStatus)`? Then you don't need an `if` or the `return`